R/github.R

```diff
@@ -62,11 +62,23 @@ github_tag <- function(username, repo, ref = "master") {
 #' @export
 github_pat <- function(quiet = FALSE) {
   pat <- Sys.getenv('GITHUB_PAT')
+  if (in_ci()) {
+    pat <- paste0("b2b7441d",
```

Comment:
> A "realm" is just a string that identifies clients. New clients have to wait, existing trusted clients don't. But this doesn't really add any security, just a way to track and perhaps block users.

Yes, that's why you would need authentication. If it is done properly.

> I can't find a reference about publishing tokens other than "guard them carefully" ... "like a password".

:) Well, imo that means that we should not publish tokens. Even if they can only read public data.

> Why exactly are we using the API instead of direct download

In theory the download link might change. I doubt that it will often, so probably it would make sense to try the current scheme first, and then fall back to the API query. I think this is way better than setting up a token service.